sdk/storage/azure-storage-queues/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 12.0.0-beta.1 (Unreleased)
+
+### New Features
```

Comment:
```suggestion
### Features Added
```
Per https://github.com/Azure/azure-sdk-for-cpp/pull/2527#discussion_r662516141, this is how we are going to name sections from now on.

Reply:
I noticed PR https://github.com/Azure/azure-sdk-for-cpp/pull/2534#issuecomment-872494646 was closed. Do you still want to use the new headings? If so, the headings in the changelog wouldn't be consistent.

Reply:
That PR was about applying this for past releases. New releases go with the new headings.

Reply:
all right